javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,18 @@
+
+
+function getFullname(firstname,surname,useFormalName) {
+    if (useFormalName == true){
+        return ("lord " + firstname + surname)
+    }else{
+        return(firstname + surname);
+    }
+
+    
+}
```

Comment:
Great work!
I have a challenge for you: if the customer is female and she decides to be called in a formal way, she will be called Lord.
What would you do to fix this?